docs: refer to the web app as "W&B" instead of "W&B App" (#2737)
wandb/docs: models/app/console-logs.mdx

keywords: ["run logs", "wandb logging", "debug runs", "log filtering", "stdout capture"]
---

When you run an experiment, you might notice messages printed to your console. W&B captures console logs and displays them in W&B. Use these messages to debug and monitor the behavior of your experiment.

The following sections describe how to view, configure, search, filter, download, and copy console logs for your runs.

## View console logs

Access console logs for a run in W&B to inspect messages produced during the run.

1. Navigate to your project in W&B.
2. Select a run within the **Runs** table.
3. Click the **Logs** tab in the project sidebar.

<Note>

> W&B stores a maximum of 100,000 lines of your logs for a run. In W&B, a maximum of 10,000 lines of your logs display at once. To view all stored lines, scroll through the logs to display older lines.

Reviewer: Nit but maybe rewrite this to avoid having "W&B" followed by "In W&B...."

## Download console logs

> To save logs locally for offline analysis or sharing, download console logs for a run in W&B:

Reviewer: ```suggestion
Download console logs for a run for offline analysis or sharing:
```
I wonder if an agent could look for and fix similar redundancies.

1. Navigate to your project in W&B.
2. Select a run within the **Runs** table.
3. Click the **Logs** tab in the project sidebar.
4. Click the download button on the console logs page.

## Copy console logs

> To paste logs into another tool or message, copy console logs for a run in W&B:

Reviewer: ```suggestion
Copy console logs for a run to paste them into a message or third-party tool:
```
